swift_browser_ui_frontend/src/entries/index_app_factory.js

```diff
@@ -0,0 +1,40 @@
+import { createApp } from "vue";
+
+import { i18n } from "@/common/i18n";
+
+import { applyPolyfills, defineCustomElements } from "csc-ui/dist/loader";
+import { vControl } from "@/common/csc-ui-vue-directive";
+
+import CFooter from "@/components/CFooter.vue";
+import LanguageSelector from "@/components/CLanguageSelector.vue";
+
+applyPolyfills().then(() => {
+  defineCustomElements();
+});
+
+export function newApp(name, data, Component) {
+  return createApp({
+    name: name,
+    components: {
+      CFooter,
+      LanguageSelector,
+    },
+    data: data,
+    created() {
+      document.title = this.$t("message.program_name");
+    },
+    methods: {
+      setCookieLang: function() {
+        const expiryDate = new Date();
+        expiryDate.setMonth(expiryDate.getMonth() + 1);
+        document.cookie = "OBJ_UI_LANG=" +
+                          i18n.locale +
+                          "; path=/; expires="
+                          + expiryDate.toUTCString();
+      },  
+    },
+    ...Component,
+  })
+    .use(i18n)
+    .directive("csc-model", vControl);
+}
```

Comment:
I followed the `csc-ui` instruction and got it worked this way:
- replacing `csc-model` to `vControl` from `index_app_factory.js`
- adding `v-control` and replacing `v-csc-model` to `v-model` like the example
<br/>

![Screenshot 2023-03-17 at 12 29 30](https://user-images.githubusercontent.com/23532204/225880490-f8e4a25f-85b3-4279-a03e-1e0feb150058.png)

Reply:
Well, that sounds easy ^^

thank you